cmd/bucket-notification-handlers.go

```diff
@@ -149,17 +171,15 @@ func (api objectAPIHandlers) PutBucketNotificationHandler(w http.ResponseWriter,
 		return
 	}
 
-	var config *event.Config
-	config, err = event.ParseConfig(io.LimitReader(r.Body, r.ContentLength), globalServerRegion, globalNotificationSys.targetList)
+	lreader := io.LimitReader(r.Body, r.ContentLength)
+	config, err := event.ParseConfig(lreader, globalServerRegion, globalNotificationSys.targetList)
 	if err != nil {
 		apiErr := errorCodes.ToAPIErr(ErrMalformedXML)
 		if event.IsEventError(err) {
 			apiErr = toAPIError(ctx, err)
 		}
-		if _, ok := err.(*event.ErrARNNotFound); !ok {
-			writeErrorResponse(ctx, w, apiErr, r.URL, guessIsBrowserReq(r))
-			return
-		}
+		writeErrorResponse(ctx, w, apiErr, r.URL, guessIsBrowserReq(r))
```

Comment:
Previously we skipped the ARN Not Found error because `event.ParseConfig` would throw `ARN Not Found` even if one of the ARN is removed in the config.

Reply:
This is reverting the fix for https://github.com/minio/minio/issues/7650

Reply:
It is not reverting the fix instead it is fixing it correctly the current master after the fix silently allows setting unconfigured arns which is a serious bug.